README.md

```diff
@@ -482,7 +482,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [CC1101]: https://crates.io/crates/cc1101
 [MPU9250]: https://crates.io/crates/mpu9250
 [BMP280]: https://crates.io/crates/bmp280-ehal
-[LSM303C_2]: https://crates.io/crates/lsm303c
+[LSM303C]: https://crates.io/crates/lsm303c
```

Comment:
@therealprof @rahul-thakoor I suggest we sort this section alphabetically for easier navigation and dedupe?

Reply:
@little-arhat You changed LSM303C here, but above it is still LSM303C_2, I think they should be in sync.

Reply:
> I suggest we sort this section alphabetically for easier navigation and dedupe?

I agree but let's do that separately.

Reply:
Yes, of course separately. Not in this PR.